File: ar/py_grpc-template/src/service/health.py
import health_pb2
import health_pb2_grpc


class HealthApi(health_pb2_grpc.HealthServiceServicer):
    async def Ping(self, request, context):
        return health_pb2.Response(message='Pong')

    async def Health(self, request, context):
        return health_pb2.Response(message='Ok')

File: projects/ar/go_grpc-template/pkg/api/v1/health/health.go
package health

import (
	"context"
	pbC "github.com/alejandrobernardis/bucket/ar/go_grpc-template/proto/api/v1/common"
	pb "github.com/alejandrobernardis/bucket/ar/go_grpc-template/proto/api/v1/health"
	"google.golang.org/protobuf/types/known/emptypb"
)

type apiHealth struct {
	pb.UnimplementedHealthServiceServer
}

func New() *apiHealth {
	return &apiHealth{}
}

func (x *apiHealth) Ping(_ context.Context, _ *emptypb.Empty) (*pbC.Response, error) {
	return &pbC.Response{Message: "Pong"}, nil
}

func (x *apiHealth) Health(_ context.Context, _ *emptypb.Empty) (*pbC.Response, error) {
	return &pbC.Response{Message: "Health"}, nil
}
